Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCustomPaletteBase.cs

```diff
@@ -3028,7 +3034,25 @@ private void ImportFromXmlDocument(XmlDocument doc)
 
                 if (version < CURRENT_PALETTE_VERSION)
                 {
-                    throw new ArgumentException($"Version '{version}' number is incompatible, only version {CURRENT_PALETTE_VERSION} or above can be imported.\nUse the PaletteUpgradeTool from the Application tab of the KryptonExplorer to upgrade.");
+                    //throw new ArgumentException($"Version '{version}' number is incompatible, only version {CURRENT_PALETTE_VERSION} or above can be imported.\nUse the PaletteUpgradeTool from the Application tab of the KryptonExplorer to upgrade.");
+
+                    DialogResult result = KryptonMessageBox.Show(
```

Comment:
I do not think it should do this, otherwise, every time a customer app is started it will be blocked until all this is run through.
The experience should be-
Load "Old xml file"
If version is "Old", run auto in memory upgrade.
If exception, then Messagebox and switch theme to default blue.

Therefore the customer user app will always run as expected, without the ned for weird Saving reloading. (i.e. the xml might be embedded and cannot be loaded !)

Reply:
Or just use the 'new' KMbox link label feature to redirect the user to the PD installer download?

Reply:
See provided animation